packages/core/src/global/global.scss

```diff
@@ -99,3 +99,20 @@ body {
     background-color: var(--tds-grey-900);
   }
 }
+
+//FIXME: In Tegel 2.0 reconsider usage of lines 68 - 100 in this file.
+
+/* Overwrite of global rules that bleed in some components */
+:root,
```

Comment:
Can we not do this in each individual file instead? I think this might be make debugging harder in the future.

Reply:
I agree, however, the import of that file would have to remain at the bottom of this one as it overrides behavior specified a couple of lines ahead. In previous projects, we usually named it "_shame.scss" to reflect that we are not proud to have it and also to indicate that we need to fix it future. 

Please let me know if you think of a better name.

Reply:
Ah I see what you mean. You could set the background in the `:host` within each file, but I think you would have to throw an `!important` on it. For example: 

<img width="498" alt="Screenshot 2024-01-08 at 13 28 07" src="https://github.com/scania-digital-design-system/tegel/assets/62651103/a7c221d2-d3ea-4ba6-8270-3165df1217d7">

I guess it's personal preference if you'd rather go with your solution above and fix it for 2.0. Or with the `!important` and fix that for 2.0.

Reply:
Yeah, exactly. I was also checking the !important rule but we might forget about it, or if ever in future we need to overright it will be chaos. 
I will prepare a separate file as maybe in the future we might need some more "dirty & ugly" code we can place it there until we reach a point for Tegel 2.0, when part of the improvements will be getting rid of shame.scss - meaning fixing the cause of issues rather than patching it.